omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,96 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated :doc:`public group <public>` to host the raw data
+   underlying the publication.
+#. Add all the authors of the paper to this new public group.
+
+Data setup
+----------
+
+#. Move the original images into the dedicated group using OMERO.web or
+   :doc:`OMERO.cli </users/cli/chgrp>`. The CLI is best used where Images or
+   Datasets are cross-linked to other Datasets or Projects in the original
+   group. The command ``bin/omero chgrp Project:$ID --include Dataset,Image``
+   cuts the cross-links in the original group and preserves the
+   Project/Dataset/Image hierarchy prepared for the move by the author.
+#. If you have used OMERO.figure to create your figures for publication, you
+   can always find the original data by using the 'info' tab, as shown in the
+   :help:`OMERO.figure Help guide <figure.html#info>` (OMERO.figure supports a
+   complete figure creation workflow, including exporting figures into image
+   processing applications for final adjustments - see the
+   :help:`OMERO.figure Help guide <figure.html>` for full details).
+#. Having all the data belong to one user simplifies the UI experience for
+   public users. If necessary, ownership of data can be transferred using the
+   'Chown' privilege (see :doc:`restricted-admins` and
+   :doc:`/users/cli/chown`).
+
+Configuring OMERO.web
+---------------------
+
+#. Configure the OMERO.web filter on the public user by setting the public URL
```

Comment:
This will probably need to be formulated more precisely. @will-moore might have an idea there. I have a suspicion that what we should be saying here is something like ``Do not put any filters on the public user, if there were any previously, remove them, so that public user can see the whole webclient UI.`` Does this sound right @will-moore ?

Reply:
In the docs, it looks to me like you have to set the filter to allow the webclient

Reply:
https://docs.openmicroscopy.org/omero/5.4.1/sysadmins/config.html?highlight=get_only#omero-web-public-url-filter implies it should be set. Would it be better to add the actual required config value here instead of referring to two other docs?

Reply:
We have a few examples of functioning regex at https://docs.openmicroscopy.org/omero/5.1.4/sysadmins/public.html - which is what I'd link to.

Reply:
My understand of https://github.com/openmicroscopy/openmicroscopy/pull/5406 is that it would simplify the filter (maybe to the point where you don't need to filter anything?) but we don't have an example on that page.

Reply:
@manics - I don't think that disabled all the "download original file" (potentially DOS the server) functionality OTOMH. I added a set of regex manually to some of our systems to disable that explicitly.

Reply:
One other thing to bear in mind is the feedback we had recently (I can't find the trello card) about the web installation docs having too many links to other docs which is why I suggested embedding the filter value.

Reply:
This is a slightly different situation though - the problem is that set of docs is that we expect people to try and read about 3 pages at once and there isn't a clear path through, in *this* case, the sysadmin will need to go and read the public user docs because it isn't feasible to reproduce all of it here, and there is a clearer path.